activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/test/java/org/activiti/cloud/services/modeling/service/ProjectServiceImplTest.java

```diff
@@ -206,4 +209,20 @@ public void should_throwImportProjectException_importingInvalidProject() throws
 
         assertTrue(actualMessage.contains(expectedMessage));
     }
+
+    @Test
+    public void should_returnProject_copyingProject() {
+        String copiedProjectName = "copied-project";
+        Project projectToCopy = new ProjectImpl("id", "copied-project");
+
+        when(projectRepository.copyProject(any(), any())).thenReturn(projectToCopy);
```

Comment:
If there are no technical limitations prefer to use more specif objects rather than `any()`.

Reply:
Done, thanks